syntax = "proto3";

message IAuthResponse {
    int32 code = 1;
    int32 userId = 2;
}
